### ISSUE TITLE
CNN made using tf.keras yields different and worse accuracy if compared to the same CNN built using keras

### ISSUE THREAD

**Issue by aurelio-amerio** [tensorflow/tensorflow#39083]
**System information**
- Have I written custom code: I'm using the CNN Mnist example from the [keras documentation](https://keras.io/examples/mnist_cnn/) 
- OS Platform and Distribution: Linux (Google Colab)
- TensorFlow installed from (source or binary): Colab
- TensorFlow version: 2.2.0-rc3
- Python version: 3.6.9
- CUDA/cuDNN version: Colab gpu default
- GPU model and memory: Colab gpu default

**Describe the current behavior**
If I train a simple CNN on the mnist dataset following the [Keras Mnist CNN example](https://keras.io/examples/mnist_cnn/) I get different accuracy depending on whether I use `tf.keras` or `keras`.

**Describe the expected behavior**
I think that the accuracy should be the same.

**Standalone code to reproduce the issue**
You can find the code reproducing this possible bug in a colab notebook here: https://colab.research.google.com/drive/1bLOyOt7tJqh2m-4rNdb_t-FZC4xX7DJI

This is the first issue I open in this repository, so I hope that the information I have provided is clear enough. 
Thank you for your work with Tensorflow!

**Comment by aurelio-amerio** [comment]
Apparently it was a matter of learning rate: keras and tf.keras have different default learning rates for Adadelta (keras uses lr=1, tf 0.001), thus it is not a bug but only a slight difference in the api.
Sorry for the inconvenience...
